Dockerfile

```diff
@@ -1,12 +1,17 @@
 # Build stage uses a full golang image to build a statically linked binary
 FROM golang:1.12 AS builder
 WORKDIR /usr/src
+
+# Download and cache dependencies
+COPY go.mod go.sum ./
+RUN go mod download
+
 COPY . .
 
-RUN CGO_ENABLED=0 go build -a -installsuffix cgo -o bin/atlas-service-broker
```

Comment:
The `-a -installsuffix cgo` is no longer necessary as of Go 1.10: https://github.com/golang/go/issues/9344#issuecomment-69944514